src/Revit/RevitHyparTools/Utils.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autodesk.Revit.DB;
+using Elements.Geometry;
+
+namespace Hypar.Revit
+{
+    public static class Utils
+    {
+        public const double FT_TO_METER_FACTOR = 0.3048;
+        public static Elements.Geometry.Profile ScaleProfileFtToMeters(Elements.Geometry.Profile profile) {
+            var transform = new Elements.Geometry.Transform();
+            transform.Scale(FT_TO_METER_FACTOR);
+            profile.Transform(transform);
```

Comment:
This transforms the profile "in place" meaning that the supplied profile will be transformed. Is that the expected behavior? It seems from the rest of the method that you want to return a new transformed profile instead. In that case, I'd do `transform.OfProfile(profile)` which will return a transformed copy of the original profile.

Reply:
you are right, I don't want to modify the original transform.